src/hypercorn/trio/tcp_server.py

```diff
@@ -48,28 +49,31 @@ async def run(self) -> None:
             socket = self.stream.socket
             ssl = False
 
+        def log_handler(e: Exception) -> None:
+            if self.config.log.error_logger is not None:
+                self.config.log.error_logger.exception("Internal hypercorn error")
```

Comment:
I think you want `error_logger.error("Internal hypercorn error", exc_info=e)`, because the exception being passed in might only be a slice of the exception group that was being caught, so the implicit "pull the current exception from ambient context" logic in `error_logger.exception` might not grab the right data

Reply:
good call, done!